spec/rapis/_files/paths.rb

```diff
@@ -0,0 +1,253 @@
+           "Access-Control-Allow-Headers"=>{"type"=>"string"}})
+      end
+    end
+    amazon_apigateway_integration do
+      responses do
+        default do
+          statusCode 200
+          responseParameters(
+            {"method.response.header.Access-Control-Allow-Methods"=>"'POST,OPTIONS'",
+             "method.response.header.Access-Control-Allow-Headers"=>
+              "'Content-Type,X-Amz-Date,Authorization,X-Api-Key'",
+             "method.response.header.Access-Control-Allow-Origin"=>"'*'"})
+        end
+      end
+      requestTemplates(
+        {"application/json"=>"{\"statusCode\": 200}"})
+      passthroughBehavior "when_no_match"
+      type "mock"
+    end
+  end
+end
+item "/pets" do
+  get do
+    produces ["application/json"]
+    parameters do
+      query "type" do
+        required false
+        type "string"
+      end
+      query "page" do
+        required false
+        type "string"
+      end
+    end
+    responses do
+      code "200" do
+        description "200 response"
+        schema do
+          ref "#/definitions/Empty"
+        end
+        headers(
+          {"Access-Control-Allow-Origin"=>{"type"=>"string"}})
+      end
+    end
+    amazon_apigateway_integration do
+      responses do
+        default do
+          statusCode 200
+          responseParameters(
+            {"method.response.header.Access-Control-Allow-Origin"=>"'*'"})
+        end
+      end
+      uri "http://petstore-demo-endpoint.execute-api.com/petstore/pets"
+      passthroughBehavior "when_no_match"
+      httpMethod "GET"
+      requestParameters(
+        {"integration.request.querystring.page"=>"method.request.querystring.page",
+         "integration.request.querystring.type"=>"method.request.querystring.type"})
+      type "http"
+    end
+  end
+  post do
+    produces ["application/json"]
+    responses do
+      code "200" do
+        description "200 response"
+        schema do
+          ref "#/definitions/Empty"
+        end
+        headers(
+          {"Access-Control-Allow-Origin"=>{"type"=>"string"}})
+      end
+    end
+    amazon_apigateway_integration do
+      responses do
+        default do
+          statusCode 200
+          responseParameters(
+            {"method.response.header.Access-Control-Allow-Origin"=>"'*'"})
+        end
+      end
+      uri "http://petstore-demo-endpoint.execute-api.com/petstore/pets"
+      passthroughBehavior "when_no_match"
+      httpMethod "POST"
+      type "http"
+    end
+  end
+  options do
+    consumes ["application/json"]
+    produces ["application/json"]
+    responses do
+      code "200" do
+        description "200 response"
+        schema do
+          ref "#/definitions/Empty"
+        end
+        headers(
+          {"Access-Control-Allow-Origin"=>{"type"=>"string"},
+           "Access-Control-Allow-Methods"=>{"type"=>"string"},
+           "Access-Control-Allow-Headers"=>{"type"=>"string"}})
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Surrounding space missing for operator `=>`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Space inside { missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Space inside } missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)